### PR TITLE
Support Django 1.8 - 1.10 with Python 3.5

### PR DESCRIPTION
**Pull request by giginet**
This PR adds supported versions
- Python3.5
- Django1.8~1.10

### CODE REVIEW

**Comment by giginet** [comment]
@lambdalisue Please review it

File: tox.ini [hunk 1]
```diff
@@ -1,29 +1,30 @@
...
-    py33-django17,
-    py34-django15,
-    py34-django16,
-    py34-django17,
+    py26-django{12,13,14,15,16}
+    py27-django{12,13,14,15,16,17,18,19,110}
+    py32-django{15,16,17}
+    py33-django{15,16,17,18}
+    py34-django{15,16,17,18,19,110}
+    py35-django{18,19,110}
```
Comment (lambdalisue) [inline tox.ini]: OK. Now we are living in 2016, let's drop the followings. Please modify `README.md` as well.

- Django 1.2 ~ 1.6
- Python 2.6
- Python 3.2

https://docs.djangoproject.com/en/1.10/faq/install/#what-python-version-can-i-use-with-django

File: src/roughpages/tests/test_views.py [hunk 2]
```diff
@@ -29,15 +29,22 @@ def setUp(self):
...
-        RequestContext.assert_called_with(self.request)
-        # t.render should be called with the context
-        t.render.assert_called_with(RequestContext())
-        # HttpResponse should be initialized with the output of
-        # t.render(c)
-        HttpResponse.assert_called_with(t.render(RequestContext()))
-        # return should be response
-        self.assertEqual(r, HttpResponse())
+        import django
+        if django.VERSION >= (1, 9):
```
Comment (lambdalisue) [inline src/roughpages/tests/test_views.py]: Probably from 1.8
https://docs.djangoproject.com/en/1.8/ref/templates/upgrading/#django-template-loader

File: src/roughpages/views.py [hunk 3]
```diff
@@ -51,6 +51,11 @@ def render_roughpage(request, t):
     """
     Internal interface to the rough page view.
     """
-    c = RequestContext(request)
-    response = HttpResponse(t.render(c))
+    import django
+    if django.VERSION >= (1, 9):
```
Comment (lambdalisue) [inline src/roughpages/views.py]: From 1.8

File: .travis.yml [hunk 4]
```diff
@@ -1,7 +1,14 @@
+sudo: false
 language: python
+python:
+  - 2.6
+  - 2.7
+  - 3.3
+  - 3.4
+  - 3.5
```
Comment (lambdalisue) [inline .travis.yml]: Drop 2.6

File: README.rst [hunk 5]
```diff
@@ -31,9 +31,9 @@ django-roughpages
 Author
     Alisue <lambdalisue@hashnote.net>
 Supported python versions
-    Python 2.6, 2.7, 3.2, 3.3, 3.4
+    Python 2.6, 2.7, 3.3, 3.4, 3.5
 Supported django versions
-    Django 1.2 - 1.6 and 1.7 rc1
+    Django 1.2 - 1.10
```
Comment (lambdalisue) [inline README.rst]: Drop Python 2.6 and Django 1.2-1.6

File: setup.py [hunk 6]
```diff
@@ -45,9 +45,9 @@ def readlist(filename):
         'Programming Language :: Python :: 2.6',
```
Comment (lambdalisue) [inline setup.py]: Drop 2.6

File: tox.ini [hunk 7]
```diff
@@ -1,33 +1,40 @@
...
-    py33-django16,
-    py33-django17,
-    py34-django15,
-    py34-django16,
-    py34-django17,
+    py26-django{12,13,14,15,16}
+    py27-django{12,13,14,15,16,17,18,19,110}
+    py33-django{15,16,17,18}
+    py34-django{15,16,17,18,19,110}
+    py35-django{18,19,110}
```
Comment (lambdalisue) [inline tox.ini]: Drop Python 2.6 and Django 1.2-1.6

File: tox.ini [hunk 8]
```diff
@@ -1,33 +1,40 @@
...
+    django16: django>=1.6,<1.7
+    django17: django>=1.7,<1.8
+    django18: django>=1.8,<1.9
+    django19: django>=1.9,<1.10
+    django110: django>=1.10,<1.11
     -rrequirements-test.txt
     coverage
 commands=
-    coverage run --source=src/roughpages runtests.py []
+    django{12,13}: pip install --upgrade "django-appconf==1.0.1"
```
Comment (lambdalisue) [inline tox.ini]: Drop this line.

File: tox.ini [hunk 9]
```diff
@@ -1,33 +1,40 @@
...
+    py{33,34,35}: mkdir -p {envdir}/build
+    py{33,34,35}: cp -rf src {envdir}/build
+    py{33,34,35}: cp -rf tests {envdir}/build
+    py{33,34,35}: 2to3 --output-dir={envdir}/build/src -W -n src
+    py{33,34,35}: 2to3 --output-dir={envdir}/build/tests -W -n tests
+    py{26,27}: coverage run --source=src/roughpages runtests.py []
```
Comment (lambdalisue) [inline tox.ini]: Drop 26

**Review by lambdalisue — CHANGES_REQUESTED** [review]
Please drop Python 2.6/3.2 and Django 1.2-1.6 as well.

**Review by lambdalisue — APPROVED** [review]
LGTM

**Comment by giginet** [comment]
Thank you.
 distribute it please.

**Pull request merged**